modules/custom/az_cas/config/install/az_cas.settings.yml

```
@@ -1,2 +1,3 @@
 disable_login_form: true
+disable_admin_add_user_button: false
```

Questions to consider:
- Is this additional config setting needed or would it be better to tie this to our existing `disable_login_form` setting?
- If we want to keep the extra setting, should it be turned on by default (disable the standard "Add User" button by default)?
```suggestion
disable_admin_add_user_button: true
```